Jenkinsfiles/cbc-pod-deployment-config.yaml

```diff
@@ -1,11 +1,11 @@
-apiVersion: v1
-kind: Pod
-spec:
-  containers:
-    - name: bb2-cbc-build
-      image: "public.ecr.aws/f5g8o1y9/bb2-cbc-build:py37-an27-tf11"
-      tty: true
-      command: ["tail", "-f", "/dev/null"]
-      imagePullPolicy: Always
-  nodeSelector:
-      Agents: true
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+    - name: bb2-cbc-build
+      image: "public.ecr.aws/f5g8o1y9/bb2-cbc-build:py37-an27-tf11"
+      tty: true
+      command: ["tail", "-f", "/dev/null"]
+      imagePullPolicy: Always
+  nodeSelector:
+      Agents: true

```

Comment:
As far as I can tell this is looks like the exact same contents so I'm not sure why the diff is reported?

Reply:
> As far as I can tell this is looks like the exact same contents so I'm not sure why the diff is reported?

could be caused by CR LF stuff...

Reply:
@JFU-GIT You might want to add this to your GIT configuration:  https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

I think this would be:

```
$ git config --global core.autocrlf true
# Configure Git to ensure line endings in files you checkout are correct for Windows.
# For compatibility, line endings are converted to Unix style when you commit files.
```